test/e2e/framework/util.go

```diff
@@ -3641,6 +3641,40 @@ func RestartKubeProxy(host string) error {
 	return nil
 }
 
+func RestartKubelet(host string) error {
+	// TODO: Make it work for all providers.
+	if !ProviderIs("gce", "aws") {
+		return fmt.Errorf("unsupported provider: %s", TestContext.Provider)
+	}
+	var cmd string
+	if ProviderIs("gce") {
+		cmd = "sudo /etc/init.d/kubelet restart"
```

Comment:
oops, GCI is systemd so this doesnt work; how do we check during test if the image is GCI or debian?